cdap-app-fabric/src/test/java/co/cask/cdap/internal/provision/ProvisioningServiceTest.java

```diff
@@ -221,7 +221,10 @@ public void testClusterCreateFailure() throws Exception {
     ProgramRunId programRunId = testProvision(ProvisioningOp.Status.FAILED,
                                               new MockProvisioner.PropertyBuilder().failCreate().build()).programRunId;
 
-    provisioningService.deprovision(programRunId);
+    // TODO: CDAP-14909 once all stores are migrated update the test with transactionals
+    Runnable task = provisioningService.deprovision(programRunId);
+    task.run();
```

Comment:
I am slightly confused. It seems that the existing code never runs the tasks. So how did this test make sense or even succeed?

Reply:
The existing code runs the task. In my previous commit I removed the task.run and added it back here.

Reply:
aaahh I missed that. Makes sense now.